src/lines.jl

```diff
@@ -68,103 +68,123 @@ function graphline(g::AbstractGraph{T}, locs_x, locs_y, nodesize::Real) where {T
         j = dst(e)
         Δx = locs_x[j] - locs_x[i]
         Δy = locs_y[j] - locs_y[i]
-        d  = sqrt(Δx^2 + Δy^2)
-        θ  = atan(Δy,Δx)
+        d = sqrt(Δx^2 + Δy^2)
+        θ = atan(Δy,Δx)
         startx = locs_x[i] + nodesize*cos(θ)
         starty = locs_y[i] + nodesize*sin(θ)
-        endx  = locs_x[i] + (d-nodesize)*1.00*cos(θ)
-        endy  = locs_y[i] + (d-nodesize)*1.00*sin(θ)
+        endx = locs_x[j] + nodesize*cos(θ+π)
+        endy = locs_y[j] + nodesize*sin(θ+π)
         lines[e_idx] = [(startx, starty), (endx, endy)]
     end
     return lines
 end
 
 function graphcurve(g::AbstractGraph{T}, locs_x, locs_y, nodesize::Vector{<:Real}, arrowlength, angleoffset, outangle=pi/5) where {T<:Integer}
-    lines = Array{Vector}(ne(g))
+    curves = Array{Vector{Tuple{Real,Real}}}(undef, ne(g))
```

Comment:
Can we use `Array{Vector{Tuple{Float64, Float64}}}` here instead? Or maybe get the correct number type (let call it `FT`) from some of the arguments, so that we can write `Array{Vector{Tuple{FT,  FT}}}`

Reply:
Also, curves seems to be a Vector of Vectors, so maybe use `Vector{Vector{ ...` instead

Reply:
I think the trig functions will always return a Float, so we can have `curves = Vector{Vector{Tuple{Float64,Float64}}}(undef, ne(g))`.